MatterControlLib/EeProm/EePromMarlinWindow.cs

```diff
@@ -236,7 +236,7 @@ private void ExportSettings()
 				{
 					ActionButtonLabel = "Export EEPROM Settings".Localize(),
 					Title = "Export EEPROM".Localize(),
-					FileName = $"eeprom_settings_{base.GetSanitizedPrinterName()}.ini"
+					FileName = $"eeprom_settings_{base.GetSanitizedPrinterName()}"
```

Comment:
Why did we drop the file extension for firmware exports?

Reply:
The file extension is added automatically by SaveFileDialog. On my system, this results in a double extension on the file name.

![screenshot from 2018-10-18 11-29-08](https://user-images.githubusercontent.com/3131268/47175879-1a854080-d2c9-11e8-946a-3f4ca0723974.png)

The extension is not included anywhere else where we use SaveFileDialog. For example...
https://github.com/MatterHackers/MatterControl/blob/99dbeaa488e0c6f6b6d4798346db3371676b35ad/MatterControlLib/CustomWidgets/ExportPrintItemPage.cs#L195